packages/psnotify.vm/psnotify.vm.nuspec

```
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>psnotify.vm</id>
-    <version>0.2.4.20250219</version>
+    <version>0.2.4.20250505</version>
     <authors>alphillips-lab</authors>
-    <description>A POC tool to fight .NET anti-dumping tricks.</description>
+    <description>Psnotify is a POC tool to fight .NET anti-dumping tricks.</description>
```

The tool name is `psnotify`, please ensure to write the name exactly as the author does in the tool home page.
```suggestion
    <description>psnotify is a POC tool to fight .NET anti-dumping tricks.</description>
```